### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by ni0520** [community/community#135510] [Code Security]
Hello everyone... I'm a beginner.

**Comment by bigbangneo69** [comment]
欢迎欢迎。

**Comment by ghostinhershell** [comment]
Hi @ni0520, thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
